packages/dds/tree/src/util/utils.ts

```diff
@@ -3,18 +3,11 @@
  * Licensed under the MIT License.
  */
 
+import type { MapGetSet } from "@fluidframework/core-utils";
 import { assert } from "@fluidframework/core-utils/internal";
 import { Type } from "@sinclair/typebox";
 import structuredClone from "@ungap/structured-clone";
 
-/**
- * Subset of Map interface.
- */
-export interface MapGetSet<K, V> {
```

Comment:
it looks like these became public after the move. internal would be a better match to the previously un-exported state